src/components/Table/EditColumnsButton.test.tsx

```diff
@@ -0,0 +1,81 @@
+import { noop } from "src/utils";
+import { click, render } from "src/utils/rtl";
+import { EditColumnsButton } from "./EditColumnsButton";
+import { GridColumn } from "./GridTable";
+import { SimpleHeaderAndData } from "./simpleHelpers";
+
+type Data = { name: string | undefined; value: number | undefined };
+type Row = SimpleHeaderAndData<Data>;
+const nameColumn: GridColumn<Row> = { name: "name", header: "Name", data: ({ name }) => name };
+const valueColumn: GridColumn<Row> = { name: "value", header: "Value", canHide: true, data: ({ value }) => value };
```

Comment:
**nit:** I see why you're using `ttc` now. I think we'd be okay if you just did: `name: "Value", header: "Value"`. Rather than assuming the capitalization.